src/module-2/components/index.html

```diff
@@ -0,0 +1,505 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="css/styles.css" />
+    <link rel="stylesheet" type="text/css" href="css/typography.css" />
+    <link rel="shortcut icon" type="image/png" href="img/favicon.ico">
+
+    <title>Online Store</title>
+  </head>
+  <body>
+    <div class="container-center-horizontal">
```

Comment:
you may use semantic elements on the list page, e.g. `header`, `nav`, `aside` etc.